test/unit/math/expect_near_rel.hpp

```diff
@@ -139,9 +139,9 @@ void expect_near_rel(const std::string& msg, EigMat1&& x1, EigMat2&& x2,
     expect_near_rel(msg2, x1_eval(i), x2_eval(i), tol);
 }
 
-template <typename T1, typename T2>
-void expect_near_rel(const std::string& msg, const std::vector<T1>& x1,
-                     const std::vector<T2>& x2, double tol = 1e-8) {
+template <typename T1, typename T2, require_all_std_vector_t<T1, T2>...>
+void expect_near_rel(const std::string& msg, const T1& x1,
+                     const T2& x2, double tol = 1e-8) {
```

Comment:
Why do we need this change? Both versions of code ere exactley equivalent as far as I see.